src/test/java/com/amazon/ion/impl/IonReaderContinuableApplicationBinaryTest.java

```diff
@@ -126,6 +126,22 @@ public void basicLocalSymbols(boolean constructFromBytes) {
         );
     }
 
+    @ParameterizedTest(name = "constructFromBytes={0}")
+    @ValueSource(booleans = {true, false})
+    public void basicInlineSymbols(boolean constructFromBytes) {
+        IonReaderContinuableApplicationBinary reader = initializeReader(
+                constructFromBytes,
```

Comment:
Please change your editor settings to do a single indent level on a continuation line, which will be more consistent with the style used in the rest of this file (and most others in here, I think).